core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -51,6 +52,16 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       "-disown"
     )
 
+    conf.sslConf match {
+      case Some(ssl) => {
+        cmdToLaunch ++ Array("-internal_security_config", ssl)
```

Comment:
Yes, when connecting to an already running H2O cluster the user will have to pass `conf --spark.ext.h2o.internal_security_config=<conf_file>` option pointing to the conf file, it should work out of the box after we merge the change I pushed to H2O core. 

The problem is that in such case the user can't use our utility class which generates the config file and key stores as it has to be the same keystore as the one used in H2O.